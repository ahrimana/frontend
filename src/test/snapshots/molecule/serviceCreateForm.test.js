/* eslint no-undef: 0 */
/* eslint no-unused-expressions: 0 */
/* eslint no-sequences: 0 */
/* eslint quotes: 0 */

import React from 'react';
import CreateServiceForm from '../../../components/molecules/service/createForm';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe("Create service form(snapshot)", () => {
  const props = {
    dispatch: jest.fn(),
  };
  const setup = () => shallow(<CreateServiceForm {...props} />);
  it("Create service form render form", () => {
    const component = setup();
    expect(toJson(component)).toMatchSnapshot();
  });
});
