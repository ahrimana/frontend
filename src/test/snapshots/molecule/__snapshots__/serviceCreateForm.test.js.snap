exports[`Create service form(snapshot) Create service form render form 1`] = `
<form
  onSubmit={[Function]}>
  <div>
    <TextField
      autoFocus={true}
      disabled={false}
      floatingLabelFixed={false}
      floatingLabelText="Name"
      fullWidth={false}
      multiLine={false}
      onChange={[Function]}
      required={true}
      rows={1}
      type="text"
      underlineShow={true} />
  </div>
  <div>
    <RaisedButton
      disabled={false}
      fullWidth={false}
      label="Create"
      labelPosition="after"
      primary={false}
      secondary={false}
      type="submit" />
  </div>
</form>
`;
